optax/_src/alias.py

```diff
@@ -994,9 +994,27 @@ def noisy_sgd(
 ) -> base.GradientTransformation:
   r"""A variant of SGD with added noise.
 
-  It has been found that adding noise to the gradients can improve
-  both the training error and the generalization error in very deep networks.
-  
+  Noisy SGD is a variant of :func:`optax.sgd` that incorporates Gaussian noise
+  into the updates. It has been found that adding noise to the gradients can
+  improve both the training error and the generalization error in very deep
+  networks.
+
+  The update :math:`u_t` is modified to include this noise as follows:
+
+  .. math::
+    u_t \leftarrow -\alpha_t g_t + N(0, \sigma_t^2),
```

Comment:
Thanks for doing that!
Shouldn't it rather be the following?
`u_t \leftarrow -\alpha_t (g_t + N(0, \sigma_t^2))`

Reply:
Yes, you're right. We first add the noise to the gradient and then scale by the learning rate. I will review my more carefully in the future.